notebooks/en/semantic_cache_chroma_vector_database.ipynb

```diff
@@ -0,0 +1,1389 @@
+{
```

Comment:
<p>Use backticks "`" to type variables as code: <code>MAX_ROWS</code></p><p>Also, <code>MAX_ROWS</code> is not reused later. So maybe remove this reference, or replace with <code>MAX_NEWS</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Replaced with MAX_NEWS</p>